### ISSUE TITLE
Could not resolve "lodash-es"

### ISSUE THREAD

**Issue by abraxxa** [ltwlf/json-diff-ts#112]
I'm using your module in a quasar app that uses Vite.
When upgrading from 1.2.6 to 2.1.0 I'm getting the following error when running `quasar dev`: 

```
✘ [ERROR] Could not resolve "lodash-es"

    node_modules/json-diff-ts/lib/jsonCompare.js:1:42:
      1 │ import { chain, keys, replace, set } from 'lodash-es';
        ╵                                           ~~~~~~~~~~~

  You can mark the path "lodash-es" as external to exclude it from the bundle, which will remove
  this error.

✘ [ERROR] Could not resolve "lodash-es"

    node_modules/json-diff-ts/lib/jsonDiff.js:1:54:
      1 │ import { difference, find, intersection, keyBy } from 'lodash-es';
        ╵                                                       ~~~~~~~~~~~

  You can mark the path "lodash-es" as external to exclude it from the bundle, which will remove
  this error.
```

In commit acec91aad22003dcc6c689db52552a6192619157 you changed the `lodash-es` dependency to `lodash` although the commit message says that it was only to `updates all pacgaes to the latest version`.

Can you please look into the issue?
Thanks!

**Comment by AnthonyRuelle** [comment]
Arf, same problem here ...

**Comment by AnthonyRuelle** [comment]
Up, please ?

**Comment by AnthonyRuelle** [comment]
up @ltwlf, can you fix this please ?

**Comment by ltwlf** [comment]
Hi, should be fixed in v2.2.0
